compiler/luci/service/src/Nodes/CircleRange.cpp

```diff
@@ -24,4 +29,52 @@ luci::CircleNode *CloneNodeLet<CN::OPQR>::visit(const luci::CircleRange *)
   return _graph->nodes()->create<luci::CircleRange>();
 }
 
+namespace sinf
+{
+
+loco::TensorShape Algorithm::visit(const luci::CircleRange *node)
+{
+  loco::TensorShape output_shape;
+  output_shape.rank(1);
+
+  auto start_node = dynamic_cast<luci::CircleConst *>(node->start());
+  auto limit_node = dynamic_cast<luci::CircleConst *>(node->limit());
+  auto delta_node = dynamic_cast<luci::CircleConst *>(node->delta());
+
+  if (start_node == nullptr || limit_node == nullptr || delta_node == nullptr)
+  {
+    return output_shape;
```

Comment:
(note) this is a bit modified

* before : `own_shape` used  (unknown dim -> 1)  
* after : just forward unknown shape

Reply:
@kyeong8139  

I'm a bit confused about this PR. 

Since this PR title was "migrate" and there was a message like "Todo : support shape inference for non-const param of Range", I thought there might be a next PR to support non-const params. 

But it looks like this change already includes supporting non-const params.